lib/pbench/server/indexer.py

```diff
@@ -1193,7 +1200,7 @@ def expand_template(templ, d, run=None):
                         pass
                     else:
                         s = re.sub(m, val, s)
-                if run is not None and key == "controller_host":
+                elif key == "controller_host" and run is not None:
```

Comment:
OK, I like the `elif` here; but was this really an LGTM complaint? And if so, on what basis?

Reply:
Um, nope -- that was one of the tiny improvements that I couldn't resist.